src/api/groups.js

```diff
@@ -51,4 +51,13 @@ export const postGroup = groupObject => (
   })
 );
 
+export const deleteGroup = groupCode => (
+  makeMockRequest({
+    uri: `/api/groups/${groupCode}`,
+    method: METHODS.DELETE,
+    mockResponse: { payload: { code: 'groupCode' } },
```

Comment:
Payload is already included by makeMockRequest, please remove:

`{ payload: { code: 'groupCode' } }` -> `{ code: 'groupCode' }`